Project/ConsoleTest/Properties/AssemblyInfo.cs

```diff
@@ -34,3 +35,5 @@
 // [assembly: AssemblyVersion("1.0.*")]
 [assembly: AssemblyVersion("1.0.0.0")]
```

Comment:
Why not 2?

Reply:
because this one was created after first release